### DISCUSSION TITLE
Select tailwind config file manually [IntelliSense]

### DISCUSSION THREAD

**Discussion by ercanertan** [tailwindlabs/tailwindcss#10807] [Ideas]
I have multiple tailwind config files on my project.
I am aware that this package is required Tailwind config file named as tailwind.config.js or tailwind.config.cjs however is there any way to select tailwind config file or files by manual?

**Comment by wongjn** [comment]
Yep! The most straight-forward way is probably to use [`@config`](https://tailwindcss.com/docs/functions-and-directives#config).

**Reply by ercanertan** [reply]
Thanks for the reply, 
I noticed that if config is renamed then Tailwind CSS IntelliSense (VS code) doesn't work properly which was my point.
I ended up creating multiple tailwind.config.js under different path.
